src/style.css

```diff
@@ -58,6 +58,188 @@
     position: relative;
 }
 
+.middle-segment{
+    background-color: black;
+     height: 130vh; 
+    margin-top: -4%;
+    border-top-left-radius: 30px;
+    border-top-right-radius: 30px;
+    position: sticky;
+    z-index: 11;
+}
+
+.midsegment-top{
+    color: white;
+    display:grid ;
+    grid-template-columns: 3fr 1.5fr;
+    height: 7vh;
```

Comment:
All the heights, expect overall container like body or a tag one level below but consisting all within we can refrain using vh. Let's update them with auto height(default). To space the elements within you can either use padding or margin property.